etc/script/linux/release-bundle.txt

```diff
@@ -1350,6 +1350,7 @@ plugins/libngs_reads_classification.so
 plugins/libopencl_support.so
 plugins/liborf_marker.so
 plugins/libpcr.so
+plugins/libpcr_primer_design_for_dna_assembly.so
```

Comment:
Don't get the idea at all. If we remove these changes, we won't be able even to build a package with this plugin

Reply:
We can merge a big chunk of code into the master and this will be a big win. All next patches will be small a manageable.

To build the "release" bundle you will still need a separate branch with a very trivial changes in release-bundle.txt.
Note that release-bundle.txt is only used in the release, and not used during the build or testing process.

Reply:
Ok, I'd ask more direct - what do you want me to do exactly?

Reply:
Remove changes in release-bundle.txt from the PR - merging this PR into the master and inclusion of this plugin into a public release are 2 different stages.